config/environment.js

```diff
@@ -1,5 +1,14 @@
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return { };
+  return {
+    localizedModel: {
+      // If this is true the decorator will get the value in any language
+      // if neither the preferred language nor the fallbacks yield anything.
+      showSomething: false,
```

Comment:
How about we rename this to `allowAnyFallback` so its a bit more meaningful in the context?

Reply:
Ok

Reply:
Done